hs-bindgen/src-internal/HsBindgen/TH/Internal.hs

```diff
@@ -228,6 +228,10 @@ toBindgenConfigTH packageRoot config = do
         bindgenConfig =
           toBindgenConfig
             (toFilePath packageRoot <$> config)
+            DoNotCreateDirStructure
```

Comment:
Since you define conservative defaults anyway, we could use the defaults here, and move the comments to the definitions of the Default instances?